src/modules/InternetDevices.rb

```diff
@@ -1,116 +0,0 @@
-# encoding: utf-8
```

Comment:
I didn't find any control file (except one testing) which mention the client. I've checked mainly SLES and OpenSUSE ones. But I feel I don't have a good overview of all our products. So any ideas what should be checked are welcome.